src/site/content/en/blog/rakuten/index.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: How Rakuten 24's investment in Core Web Vitals increased revenue per visitor by 53.37% and conversion rate by 33.13%
+subhead: >
+  By measuring real users' Web Vitals, Rakuten 24 also found that a good Largest Contentful Paint (LCP) can lead to a conversion rate increase of 61.13%.
+description: >
+  By measuring real users' Web Vitals, Rakuten 24 also found that a good Largest Contentful Paint (LCP) can lead to a conversion rate increase of 61.13%.
+date: 2022-08-15
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/uB1PUfGOXOhrqUBW3tfO.jpg
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/OLBDJ1YEaVWGkR5lqNyR.jpg
+alt: A banner image with the Rakuten 24 logo and branded artwork.
+authors:
+  - linhduong
+  - shogokashiwase
+  - hayounglee
+  - ryunosukeakiba
+tags:
+  - blog
+  - case-study
+  - web-vitals
+---
+
+[Rakuten 24](https://24.rakuten.co.jp/) is an online store that collaborates with both major multinational and domestic consumer goods manufacturers to offer a great variety of daily necessities including healthcare, beverage, pet supplies, baby products, and more. This store is provided by Rakuten Group, Inc.—a global leader in internet services, and is among the top performers on their digital marketplace platform in Japan.
+
+Understanding the impact of web performance on user experience, the Rakuten 24 team has been continuously measuring, optimizing and monitoring Core Web Vitals and other metrics.
+
+As a result, **over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP)**.
```

Comment:
@malchata Shu Hirasaka contacted me via email regarding your concern about the lack of mention about CLS in the article. This is because we are still working on improving CLS so he suggested adding a sentence about it.

How about adding it here like this?
"As a result, over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP). They are still working on Cumulative Layout Shift (CLS) improvements."

If it doesn't work for you, feel free to suggest a better place to add this sentence.